app/controllers/invites_controller.rb

```diff
@@ -15,13 +15,13 @@ def accept
     if !logged_in?
       # we redirect to signup url because we are not logged in
       redirect_to signup_url( token: params[:id] )
-    elsif @invite.email.casecmp(current_user.email) == 0
+    elsif current_user && @invite.email.casecmp(current_user.email) == 0
```

Comment:
Please remove this check. Why is this necessary? We already have an if above that verifies if you are logged in.